server/src/main/java/org/spine3/server/entity/storage/Columns.java

```diff
@@ -118,11 +118,43 @@ private Columns() {
     }
 
     /**
-     * Generates the {@link Column Column} values considering the passed {@linkplain Entity entity type}
-     * indexed.
+     * Retrieves a {@link Column} instance of the given name and from the given Entity class.
+     *
+     * <p>If the given Entity class has not yet been added to the Column cache, it's added upon this
+     * operation.
+     *
+     * <p>If no column is found, an exception is thrown.
+     *
+     * @param entityClass the class containing the {@link Column} definition
+     * @param columnName  the name of the {@link Column}
+     * @return an instance of {@link Column} with the given name
+     * @throws IllegalStateException if the {@link Column} is not found
+     */
+    static Column<?> metadata(Class<? extends Entity> entityClass, String columnName) {
+        checkNotNull(entityClass);
+        checkNotNull(columnName);
+        ensureRegistered(entityClass);
+
+        final Collection<Column<?>> cachedColumns = knownEntityProperties.get(entityClass);
+        for (Column<?> column : cachedColumns) {
+            if (column.getName()
+                      .equals(columnName)) {
+                return column;
+            }
+        }
+
+        throw new IllegalStateException(
+                format("Could not find a Column description for %s.%s.",
+                       entityClass.getCanonicalName(),
+                       columnName));
+    }
+
+    /**
+     * Generates the {@link Column Column} values considering the passed
+     * {@linkplain Entity entity type} indexed.
      *
      * @param entityType indexed type of the {@link Entity}
-     * @param entity the object which to take the values from
+     * @param entity     the object which to take the values from
      * @return a {@link Map} of the {@link Column Columns}
      */
     private static Map<String, Column.MemoizedValue<?>> getStorageFields(
```

Comment:
BTW, this method should be renamed according to its definition above. And we don't have `Storage Fields` anymore.